scripts/docs/build-sources.mjs

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license Copyright (c) 2003-2025, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-licensing-options
+ */
+
+/* eslint-env node */
+
+import upath from 'upath';
+import fs from 'fs-extra';
+import { CKEDITOR5_ROOT_PATH, CKEDITOR5_COMMERCIAL_PATH } from '../constants.mjs';
+import generateCKEditor5DocsBuild from './generate-ckeditor5-docs-build.mjs';
+
+export default async function buildSources() {
+	const { version } = await fs.readJson( upath.join( CKEDITOR5_ROOT_PATH, 'package.json' ) );
+	const basePath = upath.join( CKEDITOR5_ROOT_PATH, 'build/docs/ckeditor5', version, 'assets' );
+
+	const output = getOutputFactory( basePath );
```

Comment:
This is only used once and is a one-liner, so there's no need to move the logic to a separate function.

```suggestion
	const output = path => upath.join( basePath, path );
```